### ISSUE TITLE
Add loading screen

### ISSUE THREAD

**Issue by karawoo** [Sage-Bionetworks/dccvalidator#422]
Add the fancy loading screen instead of making people see buttons they can't click on until the login dance completes: https://github.com/Sage-Bionetworks/SynapseShinyApp/pull/15

**Comment by Aryllen** [comment]
YES! Adding this as an issue for dccvalidator, as well. This would also be great on the metadata dictionary app since it looks like it's not doing anything until the table loads.

FYI, Tom is asking Verena if it's possible to make a transparent gif. Then we could use different background colors, if wanted (the color it is looks fine and Sage-y to me).
